### PR TITLE
b0.22 - e2e - Updated verification to new dict format of task-api tasks

### PR DESCRIPTION
**Pull request by maaktweluit**
```
options => app
[root] => golem
```

### CODE REVIEW

File: scripts/node_integration_tests/playbooks/base.py [hunk 1]
```diff
@@ -345,7 +349,12 @@ def on_success(result):
 
     def step_verify_output(self):
         settings = self.task_settings_dict
-        output_file_name = settings.get('name') + '*.' + self.output_extension
+        # Name has moved for task-api tasks
+        if 'golem' in settings:
+            name = settings.get('golem').get('name')
```
Comment (mfranciszkiewicz) [inline scripts/node_integration_tests/playbooks/base.py]: Blender-App results do not contain the value of `name` in the filename

File: scripts/node_integration_tests/playbooks/base.py [hunk 2]
```diff
@@ -111,7 +111,11 @@ def task_settings_dict(self) -> dict:
 
     @property
     def output_extension(self):
-        return self.task_settings_dict.get('options', {}).get('format')
+        # task-api tasks renamed options to app
+        app = self.task_settings_dict.get('options')
+        if app is None:
```
Comment (mfranciszkiewicz) [inline scripts/node_integration_tests/playbooks/base.py]: `node_integration_tests.helpers.is_task_api_task`

**Review by mfranciszkiewicz — APPROVED** [review]

File: scripts/node_integration_tests/playbooks/base.py (hunk 1 above)
Comment (maaktweluit) [reply scripts/node_integration_tests/playbooks/base.py]: good one, set it to '' now at all times and removed the comment

File: scripts/node_integration_tests/playbooks/base.py (hunk 2 above)
Comment (maaktweluit) [reply scripts/node_integration_tests/playbooks/base.py]: was looking for that, thank you!

**Pull request merged**